tests/wpt/web-platform-tests/css/css-backgrounds/background-clip/clip-rounded-corner-ref.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Filled Background with Rounded Corner</title>
+<style>
+    #a {
+        width: 100px;
+        height: 80px;
+        border: 20px blue solid;
+        border-top-right-radius: 20px;
+        background-color: red;
+        background-clip: border-box;
+    }
+    #shield {
+        position: absolute;
+        width: 30px;
+        height: 30px;
+        left: 120px;
+        top: 5px;
+        background-color: black;
+    }
+</style>
+<body>
+    <div id="a"></div>
+    <div id="shield"></div>
```

Comment:
Maybe the `shield` comment is useful here too?

Reply:
Well, I guess there's no corner to hide here, so nbd, please ignore this comment.